types/bank_transaction.yaml

```diff
@@ -0,0 +1,34 @@
+description: >-
+  Types needed for bookkeeping bank transactions
+types:
+  BankSessionToken:
+    description: >-
+      In Germany (an probably other countries) we have to provide a token
```

Comment:
maybe we don't have to specify the country. We can just say its an optional token to be included in the invoice?